.prettierignore

```diff
@@ -0,0 +1 @@
+.schema/version.schema.json
```

Comment:
Why this?

Reply:
ory/ci and the `format` make target in this repo disagree about the number of spaces to indent with 🤮